sashimi-webapp/src/logic/filemanager/data/file.js

```diff
@@ -1,13 +1,12 @@
 import storage from 'src/database/storage';
 import Folder from './folder';
+import idMap from './idmap';
```

Comment:
The reason I am using both relative import and absolute import is to distinguish between `Filemanager` components and others components imported from outside `Filemanager`. (example: `storage` from `Database` component.